KlimaDAO/klimadao
# Set optional fixed theme and hide ThemeToggle

## Description

Some pages require a fixed theme and the ThemeToggle should not be shown either.

## Test

1. Open the [preview site](https://klimadao-site-git-optional-theme-toggle-klimadao.vercel.app/) and change the theme to "light"
2. Click on "Buy" which has a fixed theme "dark"  => the site should be in dark and no ThemeToggle is shown 
3. Click on "Resources" or "Home" => theme should be "light" again (taken from the previous preferences in Step 1)
4. Change the theme to "dark"
5. Open the retirement page with [this link](https://klimadao-site-git-optional-theme-toggle-klimadao.vercel.app/retirements/0x087a7afb6975a2837453be685eb6272576c0bc06) which has a fixed theme "light" => theme should be light now and no ThemeToggle is shown 
6. Click on "Resources" or "Home" => theme should be "dark" again (taken from the previous preferences in Step 4)

## CAUTION

Remove test code from pages "buy" and "retirements" before merge !

## Related Ticket

Resolves https://github.com/KlimaDAO/klimadao/issues/474

## Checklist



- [x] Building the site with `npm run build-site` works without errors
- [x] Building the app with `npm run build-app` works without errors
- [x] I formatted JS and TS files with running `npm run format-all`
